js/src/layers/WMSLayer.js

```diff
@@ -18,6 +19,7 @@ export class LeafletWMSLayerModel extends tilelayer.LeafletTileLayerModel {
       transparent: false,
       version: '1.1.1',
       crs: null,
+      noWrap: true,
```

Comment:
If you want to add a new option, it must be written in `snake_case`

Reply:
Some map CRS might wrap around, should it be configurable from python?

Reply:
Yes it should also be added Python side

Reply:
Should this use the `o=True` on the Python side? are you guys ok with `no_wrap: true` being the default?

Reply:
> Should this use the o=True on the Python side?

Yes.

Actually this should be added to its base class `TileLayer`, and it should be false by default, so that it reflects the LeafletJS API: https://leafletjs.com/reference-1.6.0.html#gridlayer-nowrap

Reply:
cool, I just noticed that it's already present on the base class, we just needed to use it on the js counterpart. All changes should be ready in a few...